src/core_plugins/kibana/public/discover/partials/no_results.html

```diff
@@ -28,19 +36,35 @@
     </div>
 
     <div ng-show="opts.timefield">
+      <h3>
+        Expand your time range
+      </h3>
+
       <p>
-        <h3>Expand your time range</h3>
-        <p>I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the <a class="kuiButton kuiButton--primary kuiButton--small" ng-click="kbnTopNav.toggle('filter')" aria-expanded="kbnTopNav.is('filter')" aria-label="time picker" data-test-subj="discoverNoResultsTimefilter"><span aria-hidden="true" class="kuiIcon fa-clock-o"></span> time picker</a> button in the top right corner of your screen.
+        I see you are looking at an index with a date field. It is possible your query does not match anything in the current time range, or that there is no data at all in the currently selected time range. Click the button below to open the time picker. For future reference you can open the time picker by clicking on the
+        <a
+          class="kuiButton kuiButton--primary kuiButton--small"
+          ng-click="kbnTopNav.toggle('filter')"
+          kbn-accessible-click
+          aria-expanded="{{kbnTopNav.isCurrent('filter')}}"
+          aria-label="time picker"
+          data-test-subj="discoverNoResultsTimefilter"
+        >
```

Comment:
Since this link doesn't have an `href` attribute it [wouldn't get the link role by default](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings#el-a-nohref) so we should add this role manually.